edu.cuny.hunter.log.core/plugin.xml

```diff
@@ -2,4 +2,12 @@
 <?eclipse version="3.4"?>
 <plugin>
 
+   <extension
+         point="org.eclipse.ltk.core.refactoring.refactoringContributions">
+      <contribution
+            class="edu.cuny.hunter.log.core.contributions.LogRejuvenationContributions"
+            id="org.eclipse.log.rejuvenation.contribution">
+      </contribution>
+   </extension>
+
```

Comment:
What warning does this fix?

Reply:
This plug-in does not export all of its packages

Reply:
Agreed.